### DISCUSSION TITLE
error in the key chain certificate / password

### DISCUSSION THREAD

**Discussion by Ikjot21** [codemagic-ci-cd/codemagic-docs#2812] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

-

### Steps to reproduce

Unable to add certificate /Users/builder/Library/MobileDevice/Certificates/tmpdohzz4ps.p12 to keychain /Users/builder/Library/codemagic-cli-tools/keychains/05-11-24_88_3pz9v.keychain-db

### Expected results

proceed

### Actual results

failed

**Comment by helinanever** [comment]
Hi @Ikjot21 , can you please provide a build id so we can look into it?

**Comment by dam-ease** [comment]
Hello @Ikjot21,
kindly share you buildID here, you can find it under your build details page's 'ID' section. ﻿Navigate to the 'Builds' section on the left-hand menu, select the build you want to view, and click the copy icon.
